### ISSUE TITLE
Increment stake in Voting.transferStake

### ISSUE THREAD

**Issue by rbharath** [computablelabs/goest#132]
At present, the `transferStake` method in `Voting.vy` (called from `resolveChallenge` in `Listing.vy`) overwrites the stake in its destination address (see https://github.com/computablelabs/goest/blob/master/contracts/Listing.vy#L217):

```
self.stakes[addr][hash] = staked
```

This can cause errors in flow. In particular, if a maker has successfully fended off two challenges, the stake for their listing will be incorrect. This line should be changed to

```
self.stakes[addr][hash] += staked
```

**Comment by rbharath** [comment]
The line in question is: https://github.com/computablelabs/goest/blob/master/contracts/Voting.vy#L213

**Comment by rbharath** [comment]
Fixed by #144.

**Comment by rbharath** [comment]
Re-opening after rollback

**Comment by rbharath** [comment]
Re-implemented in #169
